resources/public/api/conf/6.0/bsas_trigger.yaml

```diff
@@ -35,18 +35,30 @@ post:
         schema:
           $ref: "./schemas/trigger/request.json"
```

Comment:
This needs to reference (using `oneOf`) the def1 and def2 schemas. (E.g. see [here](https://github.com/hmrc/self-assessment-bsas-api/blob/0010eec06c7823712671463354726399a6a1ce5b/resources/public/api/conf/6.0/foreign_property_accounting_adjustments_submit.yaml#L47-L49)).